python/lsst/dbserv/dbREST_v0.py

```diff
@@ -72,26 +57,23 @@ def runDbQuery1(query, optTuple=None):
                 retStr += "%s:%s " % (cursor.description[x][0], row[x])
         if fmt == "application/json":
             retStr = json.dumps(retStr)
-    cursor.close()
     return retStr
 
-def runDbQueryM(query, optTuple=None):
-    '''Runs query that returns many rows.'''
-    cursor = con.cursor()
-    if optTuple:
-        cursor.execute(query, optTuple)
-    else:
-        cursor.execute(query)
-    rows = cursor.fetchall()
+def runDbQueryM(query, optParams=None):
+    '''Runs query that returns many rows. Returns properly formatted result.'''
+    rows = db.execCommandN(query, optParams)
     fmt = request.accept_mimetypes.best_match(['application/json', 'text/html'])
     retStr = ''
     if len(rows) > 0:
         if fmt == 'text/html':
-            retStr = "<br />".join(str(r[0]) for r in rows)
+            for row in rows:
+                for c in row:
+                    if isinstance(c, int) or isinstance(c, float) or isinstance(c, long):
+                        c = str(c)
```

Comment:
Is this really needed? "%s" format implicitly does str().
`isinstance(c, int) or isinstance(c, float) or isinstance(c, long)` can be shortened to `isinstance(c, (int, float, long))`